src/v2/views/board/NewContactGroupBoard.tsx

```diff
@@ -0,0 +1,361 @@
+		client
+			.createContactGroup(titleValue, memberListEmails)
+			.then(() => {
+				createSnackbar({
+					key: new Date().toLocaleString(),
+					type: 'success',
+					label: t(
+						'board.newContactGroup.snackbar.contact_group_created',
+						'Contact group successfully created'
+					)
+				});
+				closeBoard();
+			})
+			.catch(() => {
+				createSnackbar({
+					key: new Date().toLocaleString(),
+					type: 'error',
+					label: t('label.error_try_again', 'Something went wrong, please try again')
+				});
+			});
+	}, [closeBoard, createSnackbar, memberListEmails, t, titleValue]);
+
+	const titleDescription = useMemo(() => {
+		if (titleValue.trim().length === 0) {
+			return t(
+				'board.newContactGroup.input.title_input.error.required',
+				'Group name is required, enter a name to proceed'
+			);
+		}
+		if (titleValue.length > CONTACT_GROUP_TITLE_MAX_LENGTH) {
+			return t(
+				'board.newContactGroup.input.title_input.error.max_length',
+				'Maximum length allowed is 256 characters'
+			);
+		}
+		return undefined;
+	}, [t, titleValue]);
+
+	const contactInputDescription = useMemo(() => {
+		let valid = 0;
+		let duplicated = 0;
+		let invalid = 0;
+
+		contactInputValue.forEach((value) => {
+			if (value.duplicated) {
+				duplicated += 1;
+			} else if (value.error) {
+				invalid += 1;
+			} else {
+				valid += 1;
+			}
+		});
+		if (valid > 0) {
+			return undefined;
+		}
+		if (invalid > 0 && duplicated > 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.invalid_already_present_addresses',
+				'Invalid and already present addresses'
+			);
+		}
+		if (invalid === 1 && duplicated === 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.invalid_address',
+				'Invalid address',
+				{ count: invalid }
+			);
+		}
+		if (invalid > 1 && duplicated === 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.invalid_address',
+				'Invalid addresses',
+				{
+					count: invalid
+				}
+			);
+		}
+		if (duplicated === 1 && invalid === 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.address_already_present',
+				'Address already present',
+				{
+					count: duplicated
+				}
+			);
+		}
+		if (duplicated > 1 && invalid === 0) {
+			return t(
+				'board.newContactGroup.input.contact_input.error.address_already_present',
+				'Addresses already present',
+				{
+					count: duplicated
+				}
+			);
+		}
+		return undefined;
+	}, [contactInputValue, t]);
+
+	const contactInputOnChange = (
+		newContactInputValue: Array<{
+			actions: Array<{ color: string; icon: string; id: string; type: string }>;
+			email: string;
+			error: boolean;
+			duplicated?: boolean;
+		}>
+	): void => {
+		// TODO item are filtered to be uniq, because the ContactInput filters out, dropdown duplicated, only visually
+		//  but provide that item inside onChange parameter
+		const uniqNewContactInputValue = uniqBy(newContactInputValue, 'email');
```

Comment:
Prefer using the function, because it is safer. If you type 'asd' instead of 'email' the uniqBy does not complaint, leading to potentially unwanted bugs in case of refactors
```suggestion
		const uniqNewContactInputValue = uniqBy(newContactInputValue, (value) => value.email);
```